crates/sqlbuiltins/src/functions/scalars.rs

```diff
@@ -0,0 +1,870 @@
+use datafusion::logical_expr::BuiltinScalarFunction;
+
+use crate::builtins::BuiltinFunction;
+// mostly using a macro here to preserve the formatting.
+// rustfmt will otherwise compact the lines.
+macro_rules! make_const {
+    (
+        $var_name:ident,
+        name => $name:expr,
+        example => $example:expr,
+        description => $description:expr
+    ) => {
+        const $var_name: (&str, &str, &str) = ($name, $example, $description);
+    };
+}
+
+make_const!(
+    ABS,
+    name => "abs",
+    example => "abs(-1)",
+    description => "Compute the absolute value of a number."
+);
+make_const!(
+    ACOS,
+    name => "acos",
+    example => "acos(0.5)",
+    description => "Compute the inverse cosine (arc cosine) of a number."
+);
+
+make_const!(
+    ACOSH,
+    name => "acosh",
+    example => "acosh(1)",
+    description => "Compute the inverse hyperbolic cosine of a number."
+);
+make_const!(
+    ASIN,
+    name => "asin",
+    example => "asin(0.5)",
+    description => "Compute the inverse sine (arc sine) of a number."
+);
+make_const!(
+    ASINH,
+    name => "asinh",
+    example => "asinh(1)",
+    description => "Compute the inverse hyperbolic sine of a number."
+);
+make_const!(
+    ATAN,
+    name => "atan",
+    example => "atan(1)",
+    description => "Computes the arctangent of a number."
```

Comment:
I think given the style set on line 40, this should be `arc tangent` and maybe say (inverse tangent?) and maybe the word "arc" should appear in the description on 46.